recipes/boost/all/conanfile.py

```diff
@@ -87,13 +88,50 @@ class BoostConan(ConanFile):
     }
 
     for libname in lib_list:
```

Comment:
so, we still use [hard-coded](https://github.com/conan-io/conan-center-index/blob/62860fa34b456c219bbd75728cfa4375dc79e534/recipes/boost/all/conanfile.py#L20) lib-list here?
probably, we should remove it as well

Reply:
indeed, I think it could be used from the yaml

Reply:
I don't think conan has a method to do that using external files.
The `self.export_folder` variable is not available in methods except `export`.
This is the reason why I copy the `yml`s to the source folder and copy it to the package folder in `package`.

The `yml` **must** be present in the export folder, because conan should be able to calculate the options/packageid without the sources present.

See https://github.com/conan-io/conan/issues/7283 for my conversation with @memsharded . He also suggested modifying `conandata.yml`, but that won't work on CCI.
I could always let the script modify the `conanfile.py`. But I don't like that idea.

Reply:
okay, so we need to modify hook then. @uilianries let's open an issue in hook, but this PR should be good for now until we have hook fixed. this is still an improvement IMO.